src/studio/AppTemplates/AspNet/App/Program.cs

```diff
@@ -1,135 +1,55 @@
-using System;
-using System.IO;
-using System.Reflection;
-using Altinn.App.Api.Controllers;
-using Altinn.App.Api.Filters;
-using Altinn.App.Api.Middleware;
-using Altinn.App.Core.Health;
-using Altinn.App.PlatformServices.Extensions;
-using Altinn.App.Services.Interface;
-using Altinn.Common.PEP.Authorization;
-using Altinn.Common.PEP.Clients;
-using AltinnCore.Authentication.JwtCookie;
-using Microsoft.ApplicationInsights;
-using Microsoft.ApplicationInsights.Extensibility;
-using Microsoft.AspNetCore.Authorization;
+using Altinn.App.Api.Extensions;
+using Altinn.App.Api.Helpers;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Server.Kestrel.Core;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Hosting;
-using Microsoft.IdentityModel.Tokens;
 using Microsoft.OpenApi.Models;
-using Newtonsoft.Json.Linq;
-using Swashbuckle.AspNetCore.SwaggerGen;
 
-var builder = WebApplication.CreateBuilder(args);
+void RegisterCustomAppServices(IServiceCollection services, IConfiguration config)
```

Comment:
Remembered it now as it's used in the app I am working on now - this method should have `IConfigurIWebHostEnvironment env` as parameter as well. Sometimes you need to register different implementations based on different implementations.

Reply:
IWebHostEnvironment added as param